src/evergreen/metrics/buildmetrics.py

```diff
@@ -178,13 +240,43 @@ def _percent_tasks(self, n_tasks):
             return 0
         return n_tasks / self.total_tasks
 
+    def _percent_display_tasks(self, n_tasks):
+        """
+        Calculate the percent of display n_tasks out of display total.
+
+        :param n_tasks: Number of display tasks to calculate percent of.
+        :return: percentage display n_tasks is out of total display tasks.
+        """
+        if self.total_display_tasks == 0:
+            return 0
+        return n_tasks / self.total_display_tasks
+
+    def _get_task_status_score(self, task):
```

Comment:
This seems like it could be useful on the `Task` object. Then it would be easy to use in other situations as well.